### PR TITLE
[SMALLFIX] Adding some additional diagnostic messages in tests to understand it better.

### PR DESCRIPTION
**Pull request by yuzhu**
Adding some additional diagnostic messages in tests to understand it better.

### CODE REVIEW

File: tests/src/test/java/alluxio/client/fs/ConcurrentFileSystemMasterCreateTest.java [hunk 1]
```diff
@@ -321,7 +323,12 @@ private void runLoadMetadata(WriteType writeType, boolean useSinglePath, boolean
               paths, limitMs);
     }
     if (!errors.isEmpty()) {
-      Assert.fail("Encountered " + errors.size() + " errors, the first one is " + errors.get(0));
+      StringWriter sw = new StringWriter();
+      PrintWriter pw = new PrintWriter(sw);
+      errors.get(0).printStackTrace(pw);
+
+      Assert.fail("Encountered " + errors.size() + " errors, the first one is " + errors.get(0)
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/client/fs/ConcurrentFileSystemMasterCreateTest.java]: FYI guava has a utility for this, I think it's called `Throwables.getStackTrace(Throwable t)`

Comment (aaudiber) [inline tests/src/test/java/alluxio/client/fs/ConcurrentFileSystemMasterCreateTest.java]: I think `errors.get(0).toString()` is a superset of `errors.get(0).getMessage()`

Comment (yuzhu) [reply tests/src/test/java/alluxio/client/fs/ConcurrentFileSystemMasterCreateTest.java]: oh cool. done

Comment (yuzhu) [reply tests/src/test/java/alluxio/client/fs/ConcurrentFileSystemMasterCreateTest.java]: removed `getMessage`

**Comment by yuzhu** [comment]
jenkins test this please

**Comment by gpang** [comment]
Jenkins, test this please.

**Comment by yuzhu** [comment]
Jenkins, test this please.

**Pull request merged**